mailbox/event/event-rabbitmq/src/test/java/org/apache/james/mailbox/events/RabbitMQEventBusTest.java

```diff
@@ -98,6 +98,11 @@ public void beforeEach(ExtensionContext extensionContext) throws Exception {
         @Override
         public void afterEach(ExtensionContext context) {
             super.afterEach(context);
+        }
+
+        @Override
+        public void afterAll(ExtensionContext extensionContext) {
+            super.afterAll(extensionContext);
```

Comment:
you moved stoping rabbitmq docker step to `afterAll` stage, then the extension name does not match with what it does. The reason I restart docker rabbitmq every test to ensure the test stability but It is failed.

FYI I'm also trying to improve the build time at #2391

Reply:
Ok, i use directly the RabbitMqExtension now. It will be  overriden by you commit :)